### PR TITLE
feat(publish): add helm dep update command

### PR DESCRIPTION
**Pull request by AlexanderBabel**
Hey,

I guess this should be the last PR for this topic. 😅

I just tested your current version and found out that you have made a change compared to my original PR. You removed the `helm dependency update` command.

I just added it again, because it's required when using the helm package command in combination with sub charts.

Thank you for reviewing this PR in advance!

### CODE REVIEW

**Comment by m1pl** [comment]
Hey,

I removed that on purpose. A pipeline shouldn't update dependencies. Maybe you want `helm dependency build` instead?

**Comment by AlexanderBabel** [comment]
Oh thanks. Didn't know that this option exists. I adjusted the command.

File: lib/publish.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ async function publishChart(configPath, registry, name, version) {
             const chartName = `${name}-${version}.tgz`;
             await execa(
                 'helm',
-                ['dep', 'update', configPath]
+                ['dep', 'build', configPath]
```
Comment (m1pl) [inline lib/publish.js:29]: Please write the command out (`dependency`).

**Review by m1pl — CHANGES_REQUESTED** [review]

**Comment by m1pl** [comment]
> Oh thanks. Didn't know that this option exists. I adjusted the command.

Yes. The correct workflow would be to commit the generated _Chart.lock_ file into VCS. Then `helm dependency build` would consistently install the exact same version across environments.

A developer would use `helm dependency update` and check the generated _Chart.lock_ file into VCS, to update dependencies.

**Review by m1pl — APPROVED** [review]

**Pull request merged**